package.json

```diff
@@ -37,6 +37,9 @@
     "url": "https://github.com/curveball/aws-lambda/issues"
   },
   "homepage": "https://github.com/curveball/aws-lambda#readme",
+  "dependencies": {
+    "@types/aws-lambda": "~8.10.46"
```

Comment:
This is probably a devDependency, no?

Reply:
I'm not sure.

The output of of `dist/index.d.ts` contains this.

```ts
import { Application } from '@curveball/core';
import { APIGatewayProxyHandler } from 'aws-lambda';
export default function lambdaHandler(app: Application): APIGatewayProxyHandler;
```

Is it OK to leave it up to the user to install a type dependency?

It could be an optional- or peerDependency as well.

Reply:
Im not sure either, usually all @types/* dependencies are in devDependency, but maybe it's a bit different in this case because we're re-exposing the same types.

Maybe we can start with it as a devDependency, and if this causes problems we can graduate to a full one?

Reply:
I tested with npm link. The TS build in my App definitely fails if I don't install `@types/aws-lambda` myself.

Since not everyone will be using TS, as dependency is probably not the best.

I do prefer to include this in the README then.

Reply:
Alright, that sounds like a good compromise for now =)